### DISCUSSION TITLE
Cannot access github student developer pack benefit

### DISCUSSION THREAD

**Discussion by rahulpandiyan** [community/community#65172] [GitHub Education] [answered]
### Hi everyone,

Hi github hi got verified on 25th AUG 2023 but still I'm not able to access student developer pack how any days does it take?

Congrats! Your request for @copyrahul was approved and your benefits will be available soon.

**Comment by MaroIsLife** [comment]
Normally It took me few hours after activating the github student developer pack, a purple PRO logo should be shown on your profile badges after the activation.

**Reply by rahulpandiyan** [reply]
But how many days it's already 3 days over

**Reply by Jacksx20** [reply]
It took me 3 days from submitting the application to approval

**Comment by nirav-gajera** [comment]
According to the GitHub Student Developer Pack website, it usually takes 3-5 business days for the benefits to become available after your application has been approved. However, it may take longer during peak times, such as the start of a new semester.

If it has been more than 5 business days since your application was approved, you can contact[ GitHub support](https://support.github.com) for help.

**Comment by MaroIsLife** [comment]
According to educative.io it takes from 1 to 30 days to process the application.

https://www.educative.io/github-students

**Comment by advayc** [comment]
It usually takes 72 hours to process the application. 72 hours after your application you should get a email along with the github pro badge on your profile. However during peak times it can take longer. You should get the benefits in no longer then a week.

**Comment by nguyenthanhan201** [comment]
![image](https://github.com/community/community/assets/80932055/82c2dbff-ab31-4de8-b95a-83cf5e9910f2)
 
I verified but I still can not use GitHub copilot

**Reply by pixelofapicture** [reply]
same issue, any solution ?

**Reply by samus-aran** [reply]
Hi @yaseenibnakhtar  👋

We’re happy you’re here but this is a pretty old discussion and most likely not relevant anymore. We suggest you either create a new discussion or search by last activity to get more recent answers. 👍

**Comment by SAtANOO** [comment]
<img width="830" alt="Screenshot 2024-03-20 at 12 22 54" src="https://github.com/community/community/assets/161602553/d898669f-041c-4de8-86e7-a607fb1036e8">
it's been a months i was showing like this and application got rejected what should i do now..

**Comment by samus-aran** [comment, marked as answer]
We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

We will be closing this thread now.
